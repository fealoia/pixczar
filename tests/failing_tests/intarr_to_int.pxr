Void main() {
	Int[] arr = 8;
}
