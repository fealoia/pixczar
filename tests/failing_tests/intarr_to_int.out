Fatal error: exception Failure("LHS type of Int[] not the same as RHS type of Int")
